src/login/logind-button.c

```diff
@@ -12,10 +12,28 @@
 #include "fd-util.h"
 #include "logind-button.h"
 #include "logind-dbus.h"
+#include "logind-seat-dbus.h"
 #include "missing_input.h"
 #include "string-util.h"
 
-#define CONST_MAX5(a, b, c, d, e) CONST_MAX(CONST_MAX(a, b), CONST_MAX(CONST_MAX(c, d), e))
+#define KEY_MAX_INTERESTED KEY_RESTART
```

Comment:
in our enums we usually prefix the "special" enum vlaues that aren't actually valid but denote boundaries, or invalid stuff with an "_". Now, strictly speaking the key macros here are not enums, but they are close enough, hence also give this a prefix.

also, please add a comment here explaining that KEY_RESTART has the highest value of the keys listed below